src/docs/_layouts/default.html

```diff
@@ -6,12 +6,20 @@
     <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
     <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
 
+    <link rel="icon" type="image/x-icon" href="img/favicon.ico">
+    <link rel="icon" sizes="192x192" href="img/favicon-hd.png">
+    <meta name="theme-color" content="#6ED6A0">
```

Comment:
we haven't got favicon so I added it (had to include the images in the `/docs` folder though)